reports.py

```diff
@@ -407,6 +407,33 @@ def article_by_size( self ):
 		text = display_report( self.wiki, content , 'article_by_size-desc' )
 		self.publish_report( 'article_by_size-page-title', text )
 
+	def most_watched( self ):
+		cur = self.db.cursor()
+		# query on Public domain; bjweeks, MZMcBride; 2009, 2017
+		query = """SELECT
+					wl_namespace,
+					wl_title,
+					watchers
+					FROM watchlist_count
+					JOIN page
+					ON wl_namespace = page_namespace
+					AND wl_title = page_title
+					WHERE wl_namespace mod 2 = 0
+					AND wl_namespace >= 0
+					ORDER BY watchers DESC, wl_title ASC
+					LIMIT 1000;"""
+		cur.execute(query)
+		content = []
+		content.append( ['most_watched-id', 'most_watched-namespace', 'most_watched-title', 'most_watched-watchers'] )
+		i = 1
+		for row in cur.fetchall():
+			content.append( [ i, row[0], self.linkify( row[1], row[0] ), row[2] ])
+			i += 1
+
+		# Format the data as wikitext
+		text = display_report(self.wiki, content, 'most_watched-desc')
```

Comment:
( self.wiki, content, 'most-watched-desc' )

Just trying to keep a consistent style here. :)